bin/release.js

```diff
@@ -210,57 +210,60 @@ const checkReleaseStatus = coroutine(function*() {
 
   handleSpinner.create('Checking if release already exists')
 
-  githubConnection.repos.getReleases(
-    {
+  let response
+
+  try {
+    response = yield githubConnection.repos.getReleases({
       owner: repoDetails.user,
       repo: repoDetails.repo
-    },
-    (err, response) => {
-      if (err) {
-        handleSpinner.fail("Couldn't check if release exists.")
-      }
+    })
+  } catch (err) {
+    handleSpinner.fail("Couldn't check if release exists.") // (igor) why we hide err?
```

Comment:
Well, we would have to silently log it or require `--debug` - what do you think?